app/controllers/admin/volunteers.py

```diff
@@ -18,7 +19,7 @@
 def getVolunteers(query):
     '''Accepts user input and queries the database returning results that matches user search'''
 
-    return json.dumps(searchUsers(query))
+    return json.dumps(searchUsers(query,"volunteers"))
```

Comment:
Instead of "volunteer," you should have "insideParticipant." From the changes I see in the searchUser.js, this code would run the code for outside participant.